src/Powershell/NewInfrastructureAgent.cs

```diff
@@ -42,7 +42,7 @@ protected async override Task ProcessRecordAsync()
                 (EnvironmentType)CommonParametersExtension.GetValue("EnvironmentType"), null, null, null,
                 (DeploymentTarget)CommonParametersExtension.GetValue("DeploymentTargetName"), (DirectoryInfo)CommonParametersExtension.GetValue("OutputDir"), ReplaceTokensExtension.GetTokens(), Interactive.ToBool(), 
                 (string)CommonParametersExtension.GetValue("CustomerInfrastructureName") , (string)CommonParametersExtension.GetValue("Description"), 
-                false, true, DeploymentTimeoutMinutes);
+                false, true, DeploymentTimeoutMinutes, false, false);

```

Comment:
The flags should be reused by both commands.

Reply:
postponing to 2nd phase

Reply:
Currently the flags require the terminateOtherVersions flag to be active. Since here this flag is always false, these flags are not supported as well. Moved to 2nd phase of the WI since it makes sense to make these available to the agent as well